meta-refkit-core/lib/oeqa/selftest/cases/refkit_ostree.py

```diff
@@ -70,6 +77,12 @@ def update_image(self, qemu):
         old_cwd = os.getcwd()
         server = None
         try:
+            # We need to stop the refkit-udpate systemd service before starting
+            # the HTTP server (and thus making any update available) to prevent
+            # the service from racing with us and potentially winning, doing a
+            # full update cycle including a final reboot.
+            self.stop_update_service(qemu)
```

Comment:
We could also have created the image with the update service disabled in a ROOTFS_POSTPROCESS_COMMAND, but I like this solution better because it also covers the systemctl runtime operation.